assets/sass/patterns/atoms/reference.scss

```diff
@@ -136,7 +161,7 @@ a.reference__authors_link:hover {
     display: inline;
   }
 
-  .doi + .reference__abstracts > .reference__abstract,
+  .reference__abstracts > .reference__abstract,
```

Comment:
why is this change needed?

Reply:
wouldn't this impact `.doi` links?

Reply:
This had been removed before but not sure why it returned.